modin/experimental/cloud/rayscale.py

```diff
@@ -151,7 +151,12 @@ def __make_config(self):
         res = self._update_conda_requirements(config["setup_commands"][0])
         config["setup_commands"][0] = res
 
-        return bootstrap_config(config)
+        old_setup_commands = config["setup_commands"]
+        old_head_setup_commands = config["head_setup_commands"]
+        bootstraped_config = bootstrap_config(config)
+        bootstraped_config["setup_commands"] = old_setup_commands
+        bootstraped_config["head_setup_commands"] = old_head_setup_commands
+        return bootstraped_config
```

Comment:
The problematic side effect is called by `merge_setup_commands` function that leads to the repetition of the settings commands(with multiple `bootstrap_config(config)` calls):
https://github.com/ray-project/ray/blob/77fb273c61dbab2ab2b93101e8649f4eabe7d6ad/python/ray/autoscaler/_private/util.py#L252